src/Evolution/DiscontinuousGalerkin/VariableFixing/RadiallyFallingFloor.hpp

```diff
@@ -0,0 +1,55 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"
+#include "Utilities/EqualWithinRoundoff.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace VariableFixing {
+/// \ingroup VariableFixingGroup
+/// \brief Applies a pressure and density floor dependent on the distance
+/// to the origin.
+///
+/// Applies the floors:
+/// \f$\rho(r) \geq \rho(r)_{fl} = 10^{-5)r^{-3/2}\f$
```

Comment:
This isn't rendering right

Reply:
The `fl` should be in `mathrm{}`, and why not spell it out as `floor`